### ISSUE TITLE
follow up `flutter symbolize` and `split-debug-info`

### ISSUE THREAD

**Issue by marandaneto** [getsentry/sentry-dart#546]
https://github.com/flutter/flutter/issues/59321#issuecomment-886343823

right now it's not possible to use `flutter symbolize` since the address doesn't match, `flutter symbolize` does the symbolication differently than Sentry probably.

**Comment by marandaneto** [comment]
@fzyzcjy sorry for the late reply.

I'd not say its a bug since symbolication works on Sentry, if we change that, it'll probably break it, I
'll see if we can find a way around it.

**Comment by fzyzcjy** [comment]
Actually I have found a way to work around:

experimment:

```
    #00 abs c205ee87 virt 00609e87 _kDartIsolateSnapshotInstructions+0x5fde87
    #00 abs c205ee87 virt 00609e87 (NOTE: need a trailing space here)
    #00 abs c205ee87 virt 00609e87 _kDartIsolateSnapshotInstructions+0x609e87
    #00 abs 0 virt 00609e87 (NOTE: need a trailing space here)
```

result:

```
#0      __BodyState.build.<anonymous closure> (package:yplusplus/pages/debug_page.dart:122:34)
#1      __BodyState.build.<anonymous closure> (package:yplusplus/pages/debug_page.dart:122:34)
#2      __BodyState.build.<anonymous closure>.<anonymous closure> (package:yplusplus/pages/edit_user_metadata_page.dart:120:26)
#3      __BodyState.build.<anonymous closure> (package:yplusplus/pages/debug_page.dart:122:34)
```

conclusion: throw away the part of `_kDartIsolateSnapshotInstructions+xxx`, such that flutter symbolize will look at virt addr, which sentry gives.

**Comment by bruno-garcia** [comment]
This could be a feature where opt'ing in to `options.attachSymbolizeStacktrace` during `init` would add a `Context` called `symbolize` with the raw stacktrace to the event. 

This could help unblock iOS users #444 until Flutter's tooling supports Mach-O files.

Is this something you could contribute a PR for?

**Comment by fzyzcjy** [comment]
@bruno-garcia I do not quite get it. Currently I do attach raw stack traces and symbolize them by myself. But how would that unblock #444? IMHO it requires sentry *server* to change code

**Comment by bruno-garcia** [comment]
#444 requires changes on Flutter's tooling (to output Mach-O instead of ELF) which is eventually going to land.

You're already attaching the raw stacktrace so I assume you do that by yourself? I was suggesting doing this as part of the SDK to anyone who opt's in.

Perhaps I'm misunderstanding this issue? Is it just a duplicate of #444?

**Comment by fzyzcjy** [comment]
Yes I do that by myself. It is indeed quite easy ;)

Even if it is in sdk, people still need to manually copy that stacktrace to local computer, and use flutter symbolize command with a local symbol file to demangle it :/ so it is still not very comfortable

**Comment by marandaneto** [comment]
Working on #444
